### PR TITLE
Adaption of verapamil model and all components required for evaluation to the verapamil model published by Hanke et al. 2020

### PR DESCRIPTION
**Pull request by AnnikaRPS**

### CODE REVIEW

File: Evaluation/Input/Content/References.md [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
+
+**Blume 1994** Blume H, Mutschler E (1989) Bioäquivalenz: Qualitätsbewertung wirkstoffgleicher Fertigarzneimittel: Anleitung, Methoden, Materialien. Govi-Verlag
+
+**Blume 1987** Blume H, Mutschler E (1989) Bioäquivalenz: Qualitätsbewertung wirkstoffgleicher Fertigarzneimittel: Anleitung, Methoden, Materialien. Govi-Verlag
+
+**Blume 1989** Blume H, Mutschler E (1989) Bioäquivalenz: Qualitätsbewertung wirkstoffgleicher Fertigarzneimittel: Anleitung, Methoden, Materialien. Govi-Verlag
+
+**Blume 1990** Blume H, Mutschler E (1989) Bioäquivalenz: Qualitätsbewertung wirkstoffgleicher Fertigarzneimittel: Anleitung, Methoden, Materialien. Govi-Verlag
+
+**Blume 1994** Blume H, Mutschler E (1989) Bioäquivalenz: Qualitätsbewertung wirkstoffgleicher Fertigarzneimittel: Anleitung, Methoden, Materialien. Govi-Verlag
```
Comment (Yuri05) [inline Evaluation/Input/Content/References.md:17]: The 6 refs above seems to refer to the same publication?

File: Evaluation/Input/Content/References.md [hunk 2]
```diff
@@ -14,24 +48,56 @@
...
 
+**Pauli-Magnus 2000** Pauli-Magnus C, von Richter O, Burk O, Ziegler A, Mettang T, Eichelbaum M, Fromm MF. Characterization of the major metabolites of verapamil as substrates and inhibitors of P-glycoprotein. J Pharmacol Exp Ther. 2000 May;293(2):376-82. PMID: 10773005.
+
 **Perdaems 2010** Perdaems N, Blasco H, Vinson C, Chenel M, Whalley S, Cazade F, Bouzom F. Predictions of metabolic drug-drug interactions using physiologically based modelling: Two cytochrome P450 3A4 substrates coadministered with ketoconazole or verapamil. Clin Pharmacokinet. 2010 Apr;49(4):239-58.
 
 **PK-Sim Ontogeny Database Version 7.3** (https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/blob/38cf71b384cfc25cfa0ce4d2f3addfd32757e13b/PK-Sim%20Ontogeny%20Database%20Version%207.3.pdf)
 
+**Ratiopharm 1988** ratiopharm GmbH (2016) Fachinformation Verapamil-ratiopharm® N 40 mg / 80 mg Filmtabletten
+
+**Ratiopharm 1989** ratiopharm GmbH (2016) Fachinformation Verapamil-ratiopharm® N 40 mg / 80 mg Filmtabletten
```
Comment (Yuri05) [inline Evaluation/Input/Content/References.md:69]: again: 2 refs with the same source?

File: Evaluation/Input/evaluation_plan.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
-{
-    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/OSP_Qualification_Plan_Schema.json",
+﻿{
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v3.0/OSP_Qualification_Plan_Schema.json",
```
Comment (Yuri05) [inline Evaluation/Input/evaluation_plan.json:2]: latest release schema is 3.1
https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v3.1/OSP_Qualification_Plan_Schema.json

File: Evaluation/Input/Content/References.md (hunk 2 above)
Comment (AnnikaRPS) [reply Evaluation/Input/Content/References.md:69]: Yes, that's how it was done in Hanke et al. 2020.

File: Evaluation/Input/evaluation_plan.json (hunk 3 above)
Comment (AnnikaRPS) [reply Evaluation/Input/evaluation_plan.json]: I pushed a corrected version of the evaluation plan

File: Evaluation/Input/Content/References.md [hunk 4]
```diff
@@ -14,24 +48,56 @@
 
 **Kuepfer 2016** Kuepfer L, Niederalt C, Wendl T, Schlender JF, Willmann S, Lippert J, Block M, Eissing T, Teutonico D. Applied Concepts in PBPK Modeling: How to Build a PBPK/PD Model.CPT Pharmacometrics Syst Pharmacol. 2016 Oct;5(10):516-531.
 
+**Maeda 2011** Maeda K, Takano J, Ikeda Y, Fujita T, Oyama Y, Nozawa K, Kumagai Y, Sugiyama Y (2011) Nonlinear pharmacokinetics of oral quinidine and verapamil in healthy subjects: a clinical microdosing study. Clinical pharmacology and therapeutics 90(2):263–70
+
```
Comment (Yuri05) [inline Evaluation/Input/Content/References.md:52]: **McAllister 1982** ref below is corrupted (somehow the reference to Maeda inserted there at the end)

Comment (AnnikaRPS) [reply Evaluation/Input/Content/References.md:52]: Thanks for recognizing that. I updated the Reference file accordingly

File: Evaluation/Input/Content/Section1_Introduction.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
+Verapamil is administered as a 1:1 racemat of R- and S-verapamil which are metabolized mainly by CYP3A4 to S- and R-norverapamil. All four entities are mechanis-based inactivators of CYP3A4 and non-competitive inhibitors of P-gp. 
 
-The presented verapamil model was established using observed concentration-time profiles of more than 10 clinical studies with doses from 0.1 mg to 240 mg in different verapamil dosing schedules including multiple doses over up to 10 days and different routes of administration (intravenous, single and multiple oral administration). 
+The presented verapamil model was established using observed concentration-time profiles of more than 45 clinical studies with doses from 0.1 mg to 250 mg in different verapamil dosing schedules including multiple doses and different routes of administration (intravenous, single and multiple oral administration). In inculdes enantioselective plasma protein binding, enantioselective metabolism by CYP3A4, non-stereospecific Pgp transport, and passive glomerular filtration
```
Comment (Yuri05) [inline Evaluation/Input/Content/Section1_Introduction.md:5]: typo: **inculdes**

File: Evaluation/Input/Content/Section1_Introduction.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
 Verapamil is used for the treatment of high blood pressure, angina (chest pain from not enough blood flow to the heart), and supraventricular tachycardia.
 
-Its major metabolizing enzyme is CYP3A4, but also CY2C8 to some extent ([Tracy 1999](#5-references)). The dose- and time-dependent nonlinear behavior of verapamil is well described through implementation of the synergistic CYP3A4 mechanism-based (auto-)inactivation by verapamil. 
+Verapamil is administered as a 1:1 racemat of R- and S-verapamil which are metabolized mainly by CYP3A4 to S- and R-norverapamil. All four entities are mechanis-based inactivators of CYP3A4 and non-competitive inhibitors of P-gp.
```
Comment (Yuri05) [inline Evaluation/Input/Content/Section1_Introduction.md:3]: typo: **mechanis-based**

File: Evaluation/Input/Content/References.md [hunk 7]
```diff
@@ -14,24 +48,56 @@
...
 **Schlender 2016** Schlender JF, Meyer M, Thelen K, Krauss M, Willmann S, Eissing T, Jaehde U. Development of a Whole-Body Physiologically Based Pharmacokinetic Approach to Assess the Pharmacokinetics of Drugs in Elderly Individuals. Clin Pharmacokinet. 2016 Dec;55(12):1573-1589. 
 
 **Shand 1981** Shand DG, Hammill SC, Aanonsen L, Pritchett EL. Reduced verapamil clearance during long-term oral administration. Clin Pharmacol Ther. 1981 Nov;30(5):701-6. 
 
+**Shirasaka 2008** Shirasaka Y, Sakane T, Yamashita S. Effect of P-glycoprotein expression levels on the concentration-dependent permeability of drugs to the cell membrane. J Pharm Sci. 2008 Jan;97(1):553-65. doi: 10.1002/jps.21114. PMID: 17828734.
+
+**Sigma-Aldrich 2013** Sigma-Aldrich Inc (2013) A Case Study in SPE Method Development - Understanding the Dual Interaction Properties of Discovery DSC-SCX SPE Using Verapamil (and Metabolite) from Serum as a Test Example. https://www.sigmaaldrich.com/technical-documents/articles/reporter-eu/a-case-study-in-spe.html, accessed: 2020-02-25
```
Comment (Yuri05) [inline Evaluation/Input/Content/References.md:85]: link seems to be corrupted / does not work

Comment (AnnikaRPS) [reply Evaluation/Input/Content/References.md:85]: That's how the reference is given in Hanke et al. 2020. I am afraid I cannot find the referenced document

File: Evaluation/Input/Content/Section1_Introduction.md (hunk 5 above)
Comment (AnnikaRPS) [reply Evaluation/Input/Content/Section1_Introduction.md]: Now corrected

File: Evaluation/Input/Content/Section1_Introduction.md (hunk 6 above)
Comment (AnnikaRPS) [reply Evaluation/Input/Content/Section1_Introduction.md]: now corrected

File: Evaluation/Input/Content/References.md (hunk 7 above)
Comment (Yuri05) [reply Evaluation/Input/Content/References.md:85]: let's remove then the link completely; otherwise the link checkers will report it every time

Comment (AnnikaRPS) [reply Evaluation/Input/Content/References.md]: Done

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**